packages/vscode/src/extension.ts

```diff
@@ -84,6 +84,27 @@ export async function activate(context: ExtensionContext): Promise<void> {
     },
   }
 
+  // enable fileWatcher to watch .prisma folder inside node_modules
+  const value: string = JSON.stringify(
+    workspace.getConfiguration().get('files.watcherExclude'),
+  )
+  if (value.includes('**/node_modules/*/**')) {
+    const replacedValue = value.replace(
+      '**/node_modules/*/**',
+      // This is a way to except node_modules/.prisma folder from the exclude, see: https://github.com/microsoft/vscode/issues/869#issuecomment-630086813
+      '**/node_modules/{[^.],?[^p],??[^r],???[^i],????[^s],?????[^m]}*',
+    )
+    try {
+      await workspace
+        .getConfiguration()
```

Comment:
There are two instances of `workspace.getConfiguration()` in short succession. Do you think it would make sense to put that in a variable and use that twice? 

`const config = workspace.getConfiguration()`

Reply:
yes, good idea 👍